src/cf-expandables/src/ExpandableTransition.js

```diff
@@ -117,11 +126,11 @@ function ExpandableTransition( element, classes ) {
   this.collapse = collapse;
   this.expand = expand;
 
+  // Public static properties.
+  this.CLASSES = CLASSES;
+
   return this;
 }
 /* eslint-enable max-lines-per-function */
 
-// Public static properties.
-ExpandableTransition.CLASSES = CLASSES;
```

Comment:
Why delete this in favor of `this.CLASSES = CLASSES`? It seems like `ExpandableTransition.CLASSES = CLASSES` would be preferable since it adds it to the class, not to instances of the class.

Reply:
To pass the individual instance's classes on to Expandable here https://github.com/cfpb/capital-framework/pull/840/files#diff-6a2d6a4799b49a1f411abd3797d2bc18R58

Reply:
Ohhhh, duh, thank you.